geoblacklight-schema.json

```diff
@@ -0,0 +1,173 @@
+
+        },
+        "layer_modified_dt": { 
+            "type": "string", 
+            "description": " Last modification date for the metadata record in the Solr date/time format. Example: 2014-04-30T13:48:51Z. Note that this field is indexed as a Solr date-time field."
+
+        },
+        "layer_slug_s": { 
+            "type": "string", 
+            "description": " Unique identifier visible to the user, used for Permalinks. Example: stanford-vr593vj7147."
+
+        },
+        "solr_bbox": { 
+            "type": "string", 
+            "description": "Derived from georss_box_s. Bounding box as maximum values for W S E N. Example: 76.76 12.62 84.76 19.91. Note that this field is indexed as a Solr spatial field."
+
+        },
+        "solr_geom": { 
+            "type": "string", 
+            "description": "Derived from georss_polygon_s or georss_box_s. Shape of the layer as a ENVELOPE WKT using W E N S. Example: ENVELOPE(76.76, 84.76, 19.91, 12.62). Note that this field is indexed as a Solr spatial field."
+
+        },
+        "solr_year_i": { 
+            "type": "integer", 
+            "description": "Derived from dct_temporal_sm. Year for which layer is valid and only a single value. Example: 1989. Note that this field is indexed as a Solr numeric field."
+
+        },
+        "dc_creator_sm": { 
+            "type": "array",
+            "items": {
+                 "type": "string"
+             },
+            "description": "Author(s). Example: George Washington, Thomas Jefferson."
+
+        },
+        "dc_format_s": { 
+            "type": "string", 
+            "description": "File format for the layer. Examples: Shapefile, GeoTIFF, ArcGRID."
+
+        },
+        "dc_language_s": { 
+            "type": "string", 
+            "description": "Language for the layer. Example: English."
+
+        },
+        "dc_publisher_s": { 
+            "type": "string", 
+            "description": "Publisher. Example: ML InfoMap."
+
+        },
+        "dc_subject_sm": { 
+            "type": "array",
+            "items": {
+                 "type": "string"
+             },
+            "description": "Subjects, preferrably in a controlled vocabulary. Examples: Census, Human settlements."
+
+        },
+        "dc_type_s": { 
+            "type": "string",
+            "enum": ["Dataset", "Image", "PhysicalObject"],
+            "description": "Resource type."
+
+        },
+        "dct_spatial_sm": { 
+            "type": "array",
+            "items": {
+                 "type": "string"
+             },
+            "description": "Spatial coverage and place names. Example: Paris, France."
+
+        },
+        "dct_temporal_sm": { 
+            "type": "array", 
+            "items": {
+                 "type": "string"
+             },
+            "description": "Temporal coverage, typically years or dates. Example: 1989, 2010."
+
+        },
+        "dct_issued_dt": { 
```

Comment:
I believe this could be further refined by adding a `"format": "date-time"`. (See: http://json-schema.org/latest/json-schema-validation.html#anchor108)